src/dotnet/CarbonAware.Plugin.Registration/Configuration/PluginType.cs

```diff
@@ -0,0 +1,8 @@
+namespace CarbonAware.Plugin.Configuration;
+
+public enum PluginType
+{
+    None,
```

Comment:
maybe we don't need this. to keep it simple for now.

Reply:
With .net enums, having a default value that has no meaning is appropriate, which is what None will be.  If none is excluded, default behavior will be assumed.  This should be validated and an exception thrown if the default value of None is still set after configuration.

Reply:
Leaving for now and we can remove later, just to have a default